frameworks/digital-outcomes-and-specialists-5/questions/brief-responses/availability.yml

```diff
@@ -10,13 +10,10 @@ question_advice: >
   {%- else %}
   participants: {{ brief.researchDates }}
   {%- endif %}
-type: text
-smaller: true
+type: date
```

Comment:
Looking at other DOS 5 questions with this type, is the hint still correct?

Reply:
Oh good spot it's not